akka-stream/src/main/scala/akka/stream/impl/FanoutProcessor.scala

```diff
@@ -16,13 +16,16 @@ import org.reactivestreams.Subscriber
     val maxBufferSize: Int,
     val initialBufferSize: Int,
     self: ActorRef,
-    val pump: Pump)
+    val pump: Pump,
+    var subscribed: Boolean = false)
```

Comment:
there is only one implementation of this `FanoutOutputs` so I think we can remove the default param value, but I also think it would be better to remove the parameter and make the `var` private with a public getter.